FirebaseAdmin/FirebaseAdmin/Messaging/FirebaseMessagingClient.cs

```diff
@@ -251,6 +294,31 @@ private static void AddCommonHeaders(HttpRequestMessage request)
             return batch;
         }
 
+        private async Task<SendResponse> SendAsyncForSendResponse(
+            Message message,
+            bool dryRun = false,
+            CancellationToken cancellationToken = default(CancellationToken))
+        {
+            try
+            {
+                var messageId = await this.SendAsync(message, dryRun, cancellationToken).ConfigureAwait(false);
+                return SendResponse.FromMessageId(messageId);
+            }
+            catch (FirebaseMessagingException e)
+            {
+                return SendResponse.FromException(e);
+            }
+            catch (HttpRequestException e)
+            {
+                return SendResponse.FromException(MessagingErrorHandler.Instance.HandleHttpRequestException(e));
+            }
+            catch (Exception e)
+            {
+                var exception = new FirebaseMessagingException(ErrorCode.Unknown, $"{e}");
```

Comment:
Is it ok to put the exception into the `SendResponse`?

Reply:
Seems like that's how we have handled exceptions in batch send. Add the responses in `var responses = new List<SendResponse>();` with `SendResponse.FromException(...` for failed once.

Reply:
Sorry for the late response! My question was about whether it's ok to put the content of the `Exception e` into the `SendResponse` via `$"{e}"`. Sorry for being unclear!